modules/xmpp/XmppConnection.js

```diff
@@ -351,6 +351,15 @@ export default class XmppConnection extends Listenable {
         return this._lastSuccessTracker.getTimeSinceLastSuccess();
     }
 
+    /**
+     * See {@link LastRequestTracker.getLastFailedMessage}.
+     *
+     * @returns {string|null}
+     */
+    getLastFailedMessage() {
+        return this._lastSuccessTracker.getLastFailedMessage();
```

Comment:
Maybe we can rename the instance variable here, so it doesn't say "success", it feels a bit counter intuitive.

Reply:
> Maybe we can rename the instance variable here, so it doesn't say "success", it feels a bit counter intuitive.

Renamed to _rawInputTracker. Also adjusted the commit message.